src/index.html

```diff
@@ -4,9 +4,419 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img
+          class="header__logo-picture"
+          src="./images/logo.png" 
+          alt="#home"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list nav__list-flexbox">
+          <li class="nav__item">
+            <a class="nav_link is-active"
+              href="#home"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+              href="#home"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#home">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+              href="#home"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#home">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#home">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#home">
+              фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#home">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__cards">
+        <div class="card catalog__card">
+          <div class="card-block"
+            data-qa="card"
+          >
```

Comment:
why do you need 2 divs for this?

Reply:
I understood. I can delete card-block and make element for card: card__ image, card__title...